addons/mail/models/mail_mail.py

```diff
@@ -292,12 +292,14 @@ def _send_prepare_values(self, partner=None):
         if partner:
             emails_normalized = tools.email_normalize_all(partner.email)
             if emails_normalized:
+                name = partner.name.replace('@', '') or "False"
```

Comment:
Don't forget to update email_formatted computation on partner

Reply:
@tde-banana-odoo Hi, why `email_formatted` part? do we really need to do that? I think only relevant change should be done when values are sent

Reply:
i will do it, i just want to know if it's absolutely necessary